protocol.cpp

```diff
@@ -780,6 +780,7 @@ int memcache_binary_protocol::parse_response(void)
 
                 m_response_len = sizeof(m_response_hdr);
                 m_last_response.clear();
+                m_last_response.set_total_len(m_response_len);
```

Comment:
where did it affect us? in the summary printed at the end of the workload?

Reply:
When updating the bytes for each op, only the request bytes were counted (because the response bytes were not updated, causing them to be zero). 
This bug affected the results summary:
The KB/sec and ops/sec results didn't make sense.
Example for SET ops: Ops/sec = 668469.91 and KB/sec = 2705779.04, with data size = 4096. Those numbers don't add up